limes-core/src/main/java/org/aksw/limes/core/measures/measure/string/KoelnPhoneticMeasure.java

```diff
@@ -0,0 +1,197 @@
+package org.aksw.limes.core.measures.measure.string;
+
+import org.aksw.limes.core.io.cache.Instance;
+import org.apache.commons.codec.language.ColognePhonetic;
+
+public class KoelnPhoneticMeasure extends StringMeasure {
+
+    public static final int codeLength = 4;
+    
+    private static boolean charAt(String s, int i, char ... attr) {
+    	if ((s.length()>i && i > -1)) {
+    		char check = s.charAt(i);
+    		for(char c : attr) {
+    			if (c == check)
+    				return true;
+    		}
+    	}
+    	return false;
+    }
+    
+    public static String getCode(String string) {
+    	
+    	ColognePhonetic co = new ColognePhonetic();
+    	
+    	return co.encode(string);
+    	
+    	/*
+    	string = string.toUpperCase();
+    	String code = "";
+    	
+    	for(int i = 0; i<string.length(); i++) {
```

Comment:
Coding styles don't match our standards. Please fix:
 1. Don't use tabs. Replace tabs with 4 spaces.
 2. Braces in control structures are mandatory

Then update the branch of your PR.

Reply:
Will be fixed in dev branch manually.